components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkConstants.java

```diff
@@ -500,6 +500,8 @@ private RequestAttribute() {
         public static final String STEP_OPTIONS = "stepOptions";
         public static final String AUTHENTICATOR_PARAMS = "authenticatorParams";
         public static final String FORCE_AUTH_PARAM = "forceAuth";
+        public static final String SUBJECT_IDENTIFIER = "subjectIdentifier";
+        public static final String SUBJECT_ATTRIBUTE_STEP = "subjectAttributeStep";
```

Comment:
Do we need to have the step word here?

 As this is under stepOptions won't the "subjectAttribute" enough?

Reply:
This has been fixed with fa17986bf0651c6fb99754285ab171f2d38f6173